lib/orbf/rules_engine/fetch_data/fetch_data_analytics.rb

```diff
@@ -0,0 +1,71 @@
+module Orbf
+  module RulesEngine
+    class FetchDataAnalytics
+      def initialize(dhis2_connection, package_arguments)
+        @package_arguments = package_arguments
+        @dhis2_connection = dhis2_connection
+      end
+
+      # TODO: SMS ask piet ;)
+      #    - handle too long urls ?
```

Comment:
Max length would be something like 2000 chars? So we might need to add a check that we're not going over that limit (this might be something that the dhis2-gem should actually do)

In case we do, we'd need to split this into multiple calls, and then glue them back together.

Reply:
I hope the client will throw an exception but I suspect it's the server config the limitation. (URI too long)

the problem the number of orgunits or the number of data elements might be big...
so except doing a nested each_slice on "both" but how to tune this to not do too much request ?

Reply:
I suspect in the first time the number of indicators will stay low and orgunits might reach 75. 
and we will get a URI too long exception. So let's try first like it is ?

Reply:
I am all for that approach, we know it's there and that it might break, let's focus on the breakage when we encounter it.